modules/registry-common-terms.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * registry/index.adoc
+
+:_content-type: REFERENCE
+[id="openshift-registry-common-terms_{context}"]
+= Common registry terms
+
+This glossary defines common terms that are used in the registry content. 
+
+container::
+Lightweight and executable images that consist software and all its dependencies. Because containers virtualize the operating system, you can run containers anywhere, from a data center to a public or private cloud to your local host.
```

Comment:
`you can run containers anywhere, from a data center to a public or private cloud to your local host.` 
Can you formalize this sentence?
Suggestion 1: `you can run containers anywhere.`
Suggestion 2: `you can run containers in data center, a public or private cloud, or your local host.` 

You can also come up with something better if you feel so.

Reply:
acked